src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -985,7 +987,11 @@ class LicenceService(
     val sentenceChanges = getSentenceChanges(sentenceDatesRequest, licenceEntity)
 
     val updatedLicenceEntity = licenceEntity.copy(
-      statusCode = if (hasOffenderResentensedWithActiveLicense(sentenceDatesRequest, licenceEntity)) INACTIVE else licenceEntity.statusCode,
+      statusCode = if (hasOffenderBeenResentencedWithActiveLicense(
```

Comment:
fixing small typo

Reply:
There's still a small typo with `license` 😉

Reply:
I changed loads of them but missed that one! fixed in later commit